randovania/gui/ui_files/preset_dread_patches.ui

```diff
@@ -116,6 +116,32 @@
           </layout>
          </widget>
         </item>
+        <item>
+         <widget class="QGroupBox" name="bosses_group">
+          <property name="title">
+           <string>Bosses</string>
+          </property>
+          <layout class="QVBoxLayout" name="verticalLayout_3">
+           <item>
+            <widget class="QCheckBox" name="consistent_raven_beak_damage_table_check">
+             <property name="text">
+              <string>Use alternate Raven Beak damage table</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="consistent_raven_beak_damage_table_label">
+             <property name="text">
+              <string>Instead of nerfing Plasma Beam, Super Missiles, and all prior weapons using the same "debuff" factor as Wave Beam/Ice Missiles, Wave Beam and Ice Missiles will have their debuff factors altered such that they do the same damage as Plasma Beam and Super Missiles do against Raven Beak in the vanilla game.</string>
```

Comment:
`Normally, Raven Beak has a high resistance to only Wave Beam and Ice Missiles. This setting decides if that resistance should be removed, added to the previous weapons or left untouched.`